crates/debugger_ui/src/debugger_panel_item.rs

```diff
@@ -94,6 +94,9 @@ impl DebugPanelItem {
                     DebugPanelEvent::Output((client_id, event)) => {
                         Self::handle_output_event(this, client_id, event, cx)
                     }
+                    DebugPanelEvent::ClientStopped(client_id) => {
+                        Self::handle_client_stopped_event(this, client_id, cx)
```

Comment:
I think it would be more readable to call the function using `this` e.g `this.handle_client_stopped_event(client_id, cx)`

Not a blocking issue for a PR though

Reply:
We can change them, will do in another pr